search/include/pcl/search/brute_force.h

```diff
@@ -53,19 +53,16 @@ namespace pcl
       using PointCloud = typename Search<PointT>::PointCloud;
       using PointCloudConstPtr = typename Search<PointT>::PointCloudConstPtr;
 
-      using IndicesPtr = shared_ptr<std::vector<int> >;
```

Comment:
Removing them would be a breaking change

Reply:
Would it really be? That is also defined [here](https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/pcl_base.h#L62) which is included in search.h

Reply:
~would~ I should have used `might`.

I don't know the discovery rules on top of my head, but in some cases, there needs to be explicit `using` invocation in order to allow the name to be searched during lookup. In those cases, a person using `DerivedClass::IndicesPtr` might get a compile error.

Reply:
Would you prefer this: `using IndicesPtr = pcl::IndicesPtr;`? Same for `IndicesConstPtr`

Reply:
Done